OrbitGl/CaptureSerializer.h

```diff
@@ -41,4 +49,50 @@ class CaptureSerializer {
   const std::string kRequiredCaptureVersion = "1.52";
 };
 
+template <class TimersIterator>
+void CaptureSerializer::Save(std::ostream& stream, const CaptureData& capture_data,
+                             TimersIterator begin, TimersIterator end) {
+  google::protobuf::io::OstreamOutputStream out_stream(&stream);
+  google::protobuf::io::CodedOutputStream coded_output(&out_stream);
+
+  CHECK(time_graph_ != nullptr);
+
+  WriteMessage(&header, &coded_output);
+
+  orbit_client_protos::CaptureInfo capture_info = GenerateCaptureInfo(capture_data);
+  WriteMessage(&capture_info, &coded_output);
+
+  // Timers
+  int timers_count = time_graph_->GetNumTimers();
```

Comment:
Maybe _timers_count_ could be provided as an argument to the method as well to remove also the call to `time_graph_->GetNumTimers();`

Reply:
Thanks, I'll change that.

Reply:
Do we need `timers_count` at all?

Reply:
I don't think so. But haven't done any experiments yet. If you like, I can also just remove it, as it would be a design flaw anyways (if there would be more timers then numtimers).